src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
```

Comment:
But having extension methods won't let users override the behavior right ? That was the feedback when we started initially hence we move to this model